File: 1.TwoSum/Ddfish/TwoSum.swift
public class TwoSum{
    static func  twoSum(nums : [Int] , target :Int) ->[Int]{
        
        for (var i = 0 ; i < nums.count - 1 ;i++){
            for(var j = i+1 ; j < nums.count ;j++){
                if(nums[i] + nums[j] == target){
                    return [nums[i],nums[j]]
                }
            }
        }
        return []
    }

}

File: 1.TwoSum/sr333444/TwoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> origin = nums;
        sort(nums.begin(), nums.end());
        for(int i=0; i<nums.size(); i++)
        {
            for(int j=i+1; j<nums.size(); j++)
            {
                if(nums[i]+nums[j]>target)
                {
                    j = nums.size();
                }
                else if(nums[i]+nums[j]==target)
                {
                    vector<int> result;
                    for(int k=0; k<origin.size(); k++)
                    {
                        if(origin[k]==nums[i])
                        {
                            result.push_back(k+1);
                            break;
                        }
                    }
                    for(int k=0; k<origin.size(); k++)
                    {
                        if(origin[k]==nums[j]&&k+1!=result[0])
                        {
                            result.push_back(k+1);
                            break;
                        }
                    }
                    if(result[0]>result[1])
                    {
                        int k =result[0];
                        result[0] = result[1];
                        result[1] = k;
                    }
                        
                    return result;
                }
                
            }
        }
    }
};